src/de/jost_net/JVerein/io/FormularAufbereitung.java

```diff
@@ -582,5 +631,112 @@ public void printAnschreiben(Spendenbescheinigung spb, String text)
       throw new RemoteException("Fehler", e);
     }
   }
-  
+
+  @SuppressWarnings("resource")
+  public void addZUGFeRD(Rechnung re, boolean mahnung) throws IOException
+  {
+    if (re.getMitgliedskontoList().size() == 0)
+      return;
+
+    String sourcePDF = f.getAbsolutePath();
+    Einstellung e = Einstellungen.getEinstellung();
+    IZUGFeRDExporter ze = new ZUGFeRDExporterFromPDFA().ignorePDFAErrors()
+        .load(sourcePDF).setProducer("JVerein")
+        .setCreator(System.getProperty("user.name"));
+
+    Invoice invoice = new Invoice()
+        // F�lligkeitsdatum
+        .setDueDate(re.getMitgliedskontoList().getLast().getDatum())
+        // Lieferdatum
```

Comment:
Ich bekomme hier eine Fehlermeldung in Eclipse, getLast() is undefined.
Ich denke, das Problem ist, dass das erst mit Java 17 eingeführt wurde. Die Eclipse settings stehen auf Java 1.7. 
Bei Hibiscus/Jameica steht, dass Java 11 vorausgesetzt wird. Damit dürften wir eigentlich auch nur Java 11 verwenden, oder liege ich falsch.
Irgendwie schaffe ich es auch nicht den Fehler weg zu bekommen wenn ich das Projekt versuche auf 17 umzustellen.

Reply:
In 17 geht es auch noch nicht sondern erst später z.B. 21.

Reply:
stimmt, ich hatte irgendwie Version 21 bei mir eingestellt. Habe jetzt den Code geändert.

Reply:
Ich glaube, es müsste mklist.size()-1 heißen

Reply:
ja, natürlich.
habe ich korrigiert